roles/rhel_subscribe/meta/main.yml

```diff
@@ -1,2 +1,2 @@
 dependencies:
-  - openshift_facts
+- { role: openshift_facts }
```

Comment:
not really related to this pr, other than it being in the changeset, but rhel_subscribe probably shouldn't depend on openshift_facts :)